kss/builder/decanter/scss/_base-elements.scss

```diff
@@ -0,0 +1,38 @@
+@charset 'UTF-8';
```

Comment:
The KSS theme has been refactored and broken up into logical new files.  Much has been stripped out and the layout has been changed to use our grid system.